qa-tests-backend/src/main/groovy/services/GraphQLService.groovy

```diff
@@ -153,6 +156,8 @@ class GraphQLService {
             CloseableHttpClient client = HttpClients
                     .custom()
                     .setSSLSocketFactory(connectionFactory)
+                    .setRetryHandler(new DefaultHttpRequestRetryHandler(3, true))
+                    .setServiceUnavailableRetryStrategy(new DefaultServiceUnavailableRetryStrategy())
```

Comment:
Let's also change this to retry 3 times instead of 1 which is the default? I think it'd also be good to bump the interval to more than 1 second as this might be unrealistic.